website/pages/en/billing.mdx

```diff
@@ -2,10 +2,32 @@
 title: Billing
 ---
 
-## Billing on Arbitrum
-
 > Invoices are generated on a weekly basis.
 
+There are two options for paying for your query fees:
+
+- [Paying with fiat currency through Banxa](#billing-on-banxa)
+- [Paying with GRT on Arbitrum](#billing-on-arbitrum)
```

Comment:
I wonder if we need "on Arbitrum" here... technically you can be on Ethereum mainnet and we'll bridge your GRT. You don't have to have Arbitrum GRT. 
Maybe we could say "Paying with crypto wallet" (cause you could use either Ethereum GRT and Arbitrum GRT)

Reply:
Good call! Tweaked it :)